src/app/api/services/dot-workflow/dot-workflow.service.ts

```diff
@@ -44,4 +44,20 @@ export class DotWorkflowService {
             })
             .pluck('entity');
     }
+
+    /**
+     * Updates the workflow actions for a page asset
+     *
+     * @param {string} inode
+     * @returns {Observable<any>} // contentlet
+     * @memberof DotWorkflowService
+     */
+    fireWorkflowAction(inode: string, actionId: string): Observable<any> {
+        return this.coreWebService
```

Comment:
I would be nice to have something such as coreWebService.put (url, body),  post(url, boy), delete(url), get(url)